numpy/core/_add_newdocs.py

```diff
@@ -6457,6 +6457,7 @@ def luf(lamdaexpr, *args, **kwargs):
 
       (dtype, offset[, title])
 
+    Offset is limited to C int, which is usually signed 32 bits.
```

Comment:
Perhaps `signed and usually 32 bits`. I think `int` is require to be signed

Reply:
fixed